CHANGES.rst

```diff
@@ -231,6 +231,12 @@ ramp_fitting
 
 - Updated the flow of the detector 1 pipeline when selecting the ``LIKELY`` algorithm
   for ramp fitting.  The ramps must contain a minimum number of groups (4).[#8631]
+  
+refpix
+------
+
+- Add optimized convolution kernel instead of the running median for NIR
+  fullframe data. [#8726]
```

Comment:
```suggestion
```

now that #8671 is merged (switching change log handling to `towncrier`) this change log entry should be a file in `changes/` instead:
```shell
echo "Add optimized convolution kernel instead of the running median for NIR fullframe data." > changes/8726.refpix.rst
```

(new PRs will include instructions on how to do this)